src/viam/sdk/tests/mocks/camera_mocks.cpp

```diff
@@ -113,60 +113,6 @@ std::shared_ptr<MockCamera> MockCamera::get_mock_camera() {
     return camera;
 }
 
-MockCameraStub::MockCameraStub() : server_(std::make_shared<CameraServer>()) {
```

Comment:
For both `camera_mocks` and `generic_mocks`, you'll see that I've removed mock client and server classes and their relevant code. From what I understand, a single test of a client method with `client_to_mock_pipeline` _should_ exercise client, server, and actual resource code, and I didn't think three separate tests (leveraging three classes like `MockCamera`, `MockCameraClient` and `MockCameraStub`) for each method was necessary. I could _totally_ be wrong in that assumption, and am happy to add this code back if desired.